### PR TITLE
fix: Tesla

### PR DESCRIPTION
**Pull request by Romeo-PHILLIPS**

### CODE REVIEW

**Comment by Romeo-PHILLIPS** [comment]
@iandees what was the reason behind #4839 and playwright?

File: locations/spiders/tesla.py [hunk 1]
```diff
@@ -4,104 +4,60 @@
...
     item_attributes = {"brand": "Tesla", "brand_wikidata": "Q478214"}
-    allowed_domains = ["www.tesla.com"]
-    start_urls = [
-        "https://www.tesla.com/findus/list",
-    ]
+
     download_delay = 0.5
     custom_settings = {
         "USER_AGENT": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36",
     }
```
Comment (Cj-Malone) [inline locations/spiders/tesla.py:18]: Is the custom useragent is still needed, can `BROWSER_DEFAULT` be used? If it's not needed, let's remove it.

Comment (Romeo-PHILLIPS) [reply locations/spiders/tesla.py:18]: Seems to be needed

**Comment by iandees** [comment]
> what was the reason behind https://github.com/alltheplaces/alltheplaces/pull/4839 and playwright

I was trying to get the Tesla spider working and tried using playwright to get real browser behavior. That didn't work, so I'll close it.

**Pull request merged**